src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralService.kt

```diff
@@ -345,9 +348,39 @@ class ReferralService(
     }
   }
 
-  fun updateDraftReferral(referral: Referral, update: DraftReferralDTO): Referral {
-    validateDraftReferralUpdate(referral, update)
+  private fun updateContainsReferralDetails(update: DraftReferralDTO): Boolean {
+    return update.completionDeadline != null || update.furtherInformation != null || update.maximumEnforceableDays != null
```

Comment:
_not a blocker, semantics_
`DraftReferralDTO` might support this by having a `details` field or function. Currently it's fairly flat, but we're trying to create a few more granular 'sections', so it could also introduce those sections

Reply:
agreed - i think i'll save this for later though to seperate out any API changes from this work.